### PR TITLE
add support for dev cert rotation

### PR DESCRIPTION
**Pull request by tkashem**

### CODE REVIEW

**Comment by tkashem** [comment]
```
$ make build
go build -mod=vendor -trimpath -ldflags "-X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.versionFromGit="v4.0.0-alpha.0-1709-g3946b53" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.commitFromGit="3946b53b0" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.gitTreeState="dirty" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.buildDate="2022-11-08T00:05:16Z" " github.com/openshift/cluster-kube-apiserver-operator/cmd/cluster-kube-apiserver-operator


$ make build_with_dev_cert_rotation
go build -mod=vendor -trimpath -ldflags "-X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.versionFromGit="v4.0.0-alpha.0-1709-g3946b53" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.commitFromGit="3946b53b0" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.gitTreeState="dirty" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/version.buildDate="2022-11-08T00:05:28Z" -X github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/certrotationcontroller.certRotationAccelerator="60"" github.com/openshift/cluster-kube-apiserver-operator/cmd/cluster-kube-apiserver-operator
```


`make build_with_dev_cert_rotation` adds `-X github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/certrotationcontroller.certRotationAccelerator="60"`

**Comment by dgrisonnet** [comment]
/lgtm

Holding to leave a chance for others to review. Feel free to remove it.
/hold

File: Makefile [hunk 1]
```diff
@@ -100,3 +100,7 @@ verify-bindata-v4.1.0: verify-apirequestcounts-crd
 .PHONY: verify-apirequestcounts-crd
 verify-apirequestcounts-crd:
 	diff -Naup $(APIREQUESTCOUNT_CRD_SOURCE) $(APIREQUESTCOUNT_CRD_TARGET)
+
+# This should be used by CI only
+build_with_dev_cert_rotation: GO_LD_EXTRAFLAGS :=-X $(GO_PACKAGE)/pkg/operator/certrotationcontroller.certRotationAccelerator="60"
```
Comment (deads2k) [inline Makefile:105]: having our nightlies rotate quickly means that every test and developer cluster we produce until we ship  rotates quickly.  Changing this means that we will no longer get any testing (CI doesn't run long enough to trigger a rotation).

**Pull request closed without merging**